src/System.Web.HttpUtility/tests/System.Web.HttpUtility.Tests.builds

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Web.HttpUtility.Tests.csproj">
+        <TestTFMs>netcoreapp1.0</TestTFMs>
+    </Project>
+    <Project Include="System.Web.HttpUtility.Tests.csproj">
```

Comment:
Why do you need both of these configs? It will build the test project twice. You should be able to just list both netcoreapp1.0 and net46 under TestTFMs on the one project.